src/mixins/map-mixin.js

```diff
@@ -439,7 +439,7 @@ export default function mapMixin(
           resetRedrawStack()
           console.log("on move event redrawall error:", error)
         })
-      } else if (_viewBoxDim !== null && layer.getState().data.length < 2) {
+      } else if (_viewBoxDim !== null && typeof layer !== "undefined" && layer.getState().data.length < 2) {
```

Comment:
i assume this means you hit a case where `layer` was undefined somewhere, resulting in an error? do you happen to recall the exact case?

Reply:
I think this was happening any time you moved the map around after the initial load... having a hard time testing with kali/mahakali being down but will look back into this once i'm up and running again

Reply:
Testing today, this doesn't seem to be a problem anymore, seems like it must have been a result of some buggy contour code earlier resulting in an undefined layer. I'm going to revert!